babel/dates.py

```diff
@@ -406,6 +408,8 @@ def get_timezone_gmt(datetime=None, width='long', locale=LC_TIME):
     hours, seconds = divmod(seconds, 3600)
     if width == 'short':
         pattern = u'%+03d%02d'
+    elif width== 'extra':                   #Condition added to take 'ZZZZZ' datetime format
+        pattern = u'%+03d:%02d'
```

Comment:
[The TR35 document on ZZZZZ](http://www.unicode.org/reports/tr35/tr35-dates.html#Time_Zone_Goals) seems to say that a zero offset should always be represented as a single "Z", not "+00:00".

Could you fix this to do that? It would probably be worth it to add a check for this just after the total `seconds` is calculated above; simply something like

```python
if width == "iso8601" and seconds == 0:
    return "Z"
```
would probably do fine.)